File: exercise/php/day06/src/Workshop/ElfWorkshop.php
<?php

declare(strict_types = 1);

namespace Workshop;

class ElfWorkshop
{
	private $taskList = [];

	public function getTaskList()
	{
		return $this->taskList;
	}

	public function addTask($task): void
	{
		if (!empty($task)) {
			$this->taskList[] = $task;
		}
	}

	public function completeTask()
	{
		if (!empty($this->taskList)) {
			return array_shift($this->taskList);
		}

	}
}

File: exercise/java/day06/src/main/java/workshop/ElfWorkshop.java
package workshop;

import java.util.ArrayList;
import java.util.List;

public class ElfWorkshop {
    private List<String> taskList;

    public ElfWorkshop() {
        this.taskList = new ArrayList<>();
    }

    public List<String> getTaskList() {
        return taskList;
    }

    public void addTask(String task) {
        if (task != null && !task.isEmpty()) {
            taskList.add(task);
        }
    }

    public String completeTask() {
        if (!taskList.isEmpty()) {
            return taskList.remove(0);
        }
        return null;
    }
}